packages/storybook/.storybook/main.js

```diff
@@ -10,29 +11,16 @@ export const stories = [
     '../src/nimble',
     '../src/spright'
 ];
-export const addons = [
-    {
-        name: '@storybook/addon-essentials',
-        options: {
-            outline: false,
-            docs: false
-        }
-    }, {
-        name: '@storybook/addon-docs',
+export const addons = [{
+        name: getAbsolutePath("@storybook/addon-docs"),
         options: {
             mdxPluginOptions: {
                 mdxCompileOptions: {
                     remarkPlugins: [remarkGfm]
                 }
             }
         }
-    },
-    '@storybook/addon-a11y',
-    '@storybook/addon-interactions',
-    '@chromatic-com/storybook',
-    '@storybook/addon-webpack5-compiler-swc',
-    'storybook-addon-pseudo-states'
-];
+    }, getAbsolutePath('@storybook/addon-a11y'), getAbsolutePath('@chromatic-com/storybook'), getAbsolutePath('@storybook/addon-webpack5-compiler-swc'), getAbsolutePath('storybook-addon-pseudo-states')];
 
 export function webpackFinal(config) {
```

Comment:
I'm assuming we swap out the webpack config. Note we need to test that this doesn't break snapshots. See discussion in https://github.com/ni/nimble/issues/2625#issue-3114591282